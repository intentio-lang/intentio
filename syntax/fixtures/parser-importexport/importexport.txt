export (
    sinus,
    cosinus,
)


import math:sin as sinus
import math:cos
import tg as tangens
import ctg
