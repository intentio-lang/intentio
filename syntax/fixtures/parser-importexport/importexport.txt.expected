[
  {
    "contents": [
      "sinus",
      "cosinus"
    ],
    "tag": "ExportDecl"
  },
  {
    "contents": {
      "contents": [
        "math",
        "sin",
        "sinus"
      ],
      "tag": "ImportQidAs"
    },
    "tag": "ImportDecl"
  },
  {
    "contents": {
      "contents": [
        "math",
        "cos"
      ],
      "tag": "ImportQid"
    },
    "tag": "ImportDecl"
  },
  {
    "contents": {
      "contents": [
        "tg",
        "tangens"
      ],
      "tag": "ImportIdAs"
    },
    "tag": "ImportDecl"
  },
  {
    "contents": {
      "contents": "ctg",
      "tag": "ImportId"
    },
    "tag": "ImportDecl"
  }
]
